### PR TITLE
Added the two new comparison algorithms to the changelog

### PR DESCRIPTION
**Pull request by JackStouffer**

### CODE REVIEW

**Comment by JackStouffer** [comment]
Ping @MartinNowak

File: changelog.dd [hunk 1]
```diff
@@ -89,6 +91,36 @@ $(LI $(LNAME2 std-algorithm-moveEmplace, A combined `moveEmplace` was added.)
     )
 )
 
+$(LI $(LNAME2 std-algorithm-isPermutation, A combined `isPermutation` was added.)
```
Comment (MartinNowak) [inline changelog.dd]: Well combined doesn't make sense here.

Comment (JackStouffer) [inline changelog.dd]: I don't know what I was thinking when I wrote this.

**Comment by MartinNowak** [comment]
Merged 5ed2368fc447225c0acceff74997a681c0cbab3b

**Pull request closed without merging**